precompiles/solo/solo.go

```diff
@@ -264,6 +295,9 @@ func (p PrecompileExecutor) sigverify(ctx sdk.Context, tx sdk.Tx, claimMsg claim
 	if err := authsigning.VerifySignature(pubkey, signerData, sig.Data, p.txConfig.SignModeHandler(), tx); err != nil {
 		return fmt.Errorf("failed to verify signature for claim tx: %w", err)
 	}
+	// increment sequence
+	_ = acct.SetSequence(acct.GetSequence() + 1)
+	p.accountKeeper.SetAccount(ctx, acct)
```

Comment:
if we increment their account sequence, does this break any semantic assumption that there's 1 tx per sequence?  (wallets are okay and check the latest number?)

Reply:
initially I wasn't incrementing here but will called out that wallet may increment its local copy of account sequence whenever it signs, instead of whenever it sent out a tx, which to me seems true empirically (with cosmos wallet like kepler etc.)

Reply:
edge question: if they claim with their _associated_ evm wallet, does their account sequence get incremented twice and is that okay? (wasn't sure if GetSequence reads from anything that might have been modified during this tx so far?)